### PR TITLE
Home task7

### PR DESCRIPTION
**Pull request by khbrn**
### What has changed:
1. Implemented a possibility to Add movie, Edit movie and Delete movie 
2. Implemented Add movie and Edit movie form validation using formik.

### CODE REVIEW

File: src/components/MovieDialog/MovieDialog.js [hunk 1]
```diff
@@ -1,50 +1,91 @@
...
+      rating: props.movie ? props.movie.vote_average : "",
+      genres: props.movie ? props.movie.genres : [],
+      runtime: props.movie ? props.movie.runtime : "",
+      overview: props.movie ? props.movie.overview : "",
+    },
+    validate,
+    onSubmit: (values) => {
+      const movieObj = convertToMovieObject(values);
+
+      if (values.dialogType === "Add Movie") {
```
Comment (Vladlan) [inline src/components/MovieDialog/MovieDialog.js]: No magic strings - move "Add Movie" to const

File: src/services/MoviesService.js [hunk 2]
```diff
@@ -41,4 +41,47 @@ export const MoviesService = {
...
+      const data = await response.json();
+      console.log(data);
+      return data;
+    } catch (error) {
+      return { error: error.response };
+    }
+  },
+  editMovie: async (movie) => {
+    try {
+      const responseJson = await fetch(fetchURL, {
```
Comment (Vladlan) [inline src/services/MoviesService.js]: just response

**Review by Vladlan — CHANGES_REQUESTED** [review]
In general LGTM except bug (see screenshot) - after deletion of a selected movie I can see it.

Screenshot 2022-07-10 at 22 45 30

File: src/services/MoviesService.js (hunk 2 above)
Comment (khbrn) [reply src/services/MoviesService.js]: done.

File: src/components/MovieDialog/MovieDialog.js (hunk 1 above)
Comment (khbrn) [reply src/components/MovieDialog/MovieDialog.js]: all done.

**Review by Vladlan — APPROVED** [review]

**Pull request merged**